### ISSUE TITLE
Feature: custom "prev" & "next" text

### ISSUE THREAD

**Issue by Anas-Nabulsi** [edenriquez/svelte-multistep-form#25]
Hello

We need to support i18n in our app, so I recommend having an option to change the default text of "prev" & "next" buttons

Thanks.

**Comment by edenriquez** [comment]
yeah sounds good, let me start working on that, I'll let you know when ready
